Source/Core/UICommon/CMakeLists.txt

```diff
@@ -5,15 +5,19 @@ set(SRCS
   USBUtils.cpp
 )
 
-if(USE_X11)
-  set(SRCS ${SRCS} X11Utils.cpp)
-endif()
-
 set(LIBS common cpp-optparse)
 if(LIBUSB_FOUND)
   set(LIBS ${LIBS} ${LIBUSB_LIBRARIES})
 endif()
 
+if(USE_X11)
+  find_package(GTK2 REQUIRED)
```

Comment:
Can't we just assume that wxWidgets on top of X11 always forwards those? Isn't that done already?